tests/webpack-test/configCases/library/issue-18932/index.js

```diff
@@ -0,0 +1,7 @@
+it("should don't have variable name conflict", function() {
```

Comment:
This should be `should not have ...`

Reply:
Nice catch, we can sync this modification back to webpack.